src/panels/config/integrations/integration-panels/ozw/ozw-config-network.ts

```diff
@@ -0,0 +1,304 @@
+  @property({ type: Object }) public hass!: HomeAssistant;
+
+  @property({ type: Object }) public route!: Route;
+
+  @property({ type: Boolean }) public narrow!: boolean;
+
+  @property({ type: Boolean }) public isWide!: boolean;
+
+  @property() public configEntryId?: string;
+
+  @property() public ozw_instance = 0;
+
+  @internalProperty() private _network?: OZWInstance;
+
+  @internalProperty() private _statistics?: OZWNetworkStatistics;
+
+  public connectedCallback(): void {
+    super.connectedCallback();
+    if (this.ozw_instance <= 0) {
+      navigate(this, "/config/ozw/dashboard", true);
+    }
+    if (this.hass) {
+      this._fetchData();
+    }
+  }
+
+  protected firstUpdated(changedProperties: PropertyValues): void {
+    super.firstUpdated(changedProperties);
+    if (this.hass) {
+      this._fetchData();
+    }
+  }
+
+  private async _fetchData() {
+    this._network = await fetchOZWNetworkStatus(this.hass!, this.ozw_instance);
+    this._statistics = await fetchOZWNetworkStatistics(
+      this.hass!,
+      this.ozw_instance
+    );
+  }
+
+  private _generateServiceButton(service: string) {
+    return html`
+      <ha-call-service-button
+        .hass=${this.hass}
+        domain="ozw"
+        service="${service}"
+      >
+        ${this.hass!.localize("ui.panel.config.ozw.services." + service)}
+      </ha-call-service-button>
+    `;
+  }
+
+  protected render(): TemplateResult {
+    return html`
+      <hass-tabs-subpage
+        .hass=${this.hass}
+        .narrow=${this.narrow}
+        .route=${this.route}
+        .tabs=${ozwTabs}
+      >
+        <ha-config-section .narrow=${this.narrow} .isWide=${this.isWide}>
+          <div slot="header">
+            ${this.hass.localize("ui.panel.config.ozw.network.header")}
+          </div>
+
+          <div slot="introduction">
+            ${this.hass.localize("ui.panel.config.ozw.network.introduction")}
+          </div>
+          ${this._network
+            ? html`
+                <ha-card class="content network-status">
+                  <div class="card-content">
+                    <div class="details">
+                      ${networkOnlineStatuses.indexOf(this._network.Status) !==
+                      -1
+                        ? html`
+                            <ha-svg-icon
+                              .path=${mdiCheckCircle}
+                              class="network-status-icon online"
+                              slot="item-icon"
+                            ></ha-svg-icon>
+                          `
+                        : networkStartingStatuses.indexOf(
+                            this._network.Status
+                          ) !== -1
+                        ? html`
+                            <ha-svg-icon
+                              .path=${mdiCircle}
+                              class="network-status-icon starting"
+                              slot="item-icon"
+                            ></ha-svg-icon>
```

Comment:
You have a lot of duplicate code now, it would be better to calculate the class and icon first and store the in a variable. Use that variable in the render.